generators/maven/templates/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
```suggestion
distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.1/apache-maven-3.8.1-bin.zip
```

Reply:
Thanks, I'll take care of it later, as soon as the behaviour is fixed

Reply:
@pascalgrimaud I commited so it uses same version as in generator